### PR TITLE
Ss 103

### PR DESCRIPTION
**Pull request by michael-lts**
**Prototype SSIO transform of one set**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/SS-103)

# What does this Pull Request do?
Prototypes XSLT transform of the harvested loebmusic set and writing to a directory for pickup by publisher

# How should this be tested? (note harvest to transform queue has issue, we will invoke the endpoint directly for now)
- Check out this branch
- on local commandline run
> curl -k -H 'Content-Type: application/json' -X POST "https://ltsds-cloud-dev-2.lib.harvard.edu:25001/jstor_transformer/do_task" -d '{"job_ticket_id":"54321","hello":"world"}'
- look on dev under /docker/jstorforum/PROCESSING/transformed/loebmusic for files (21 files)
# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests? Integration tests are showing message flow currently

# Interested parties
Tag (@ mention) interested parties

### CODE REVIEW

**Review by cgoines — APPROVED** [review]

File: supervisord.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@ logfile_maxbytes=50MB
 logfile_backups=10 
 
 [program:gunicorn]
-command=gunicorn -w 4 -c gunicorn.conf.py "app:create_app()" -b 0.0.0.0:8081 --certfile /etc/nginx/ssl/nginx.cert --keyfile /etc/nginx/ssl/nginx.key --timeout 10000
+command=gunicorn -w 4 -c gunicorn.conf.py --reload "app:create_app()" -b 0.0.0.0:8081 --certfile /etc/nginx/ssl/nginx.cert --keyfile /etc/nginx/ssl/nginx.key --timeout 10000
```
Comment (awoods) [inline supervisord.conf:9]: Using `--reload` in dev is helpful (for debugging), but we do not want this configuration going to prod.

File: tasks/tasks.py [hunk 2]
```diff
@@ -19,5 +19,5 @@
 def do_task(message):
     url = "https://localhost:8081/jstor_transformer/do_task"
     celeryapp.execute.send_task("tasks.tasks.do_task", args=[message], kwargs={}, queue=os.getenv('NEXT_QUEUE_NAME'))
-    response = http_client.post(url, json = message, verify=False)
-    return response.json()
+    #response = http_client.post(url, json = message, verify=False)
+    return None #response.json()
```
Comment (awoods) [inline tasks/tasks.py:23]: Is returning `None` temporary? ..or the designed solution approach?

File: supervisord.conf (hunk 1 above)
Comment (michael-lts) [reply supervisord.conf:9]: I should probably just remove, check in and only use locally

File: app/jstor_transformer.py [hunk 3]
```diff
@@ -29,21 +30,23 @@ def handle_errors(self, result, error_msg, exception_msg = None, set_job_failed
...
           'success': False,
           'error': None,
           'message': ''
         }
-
         #Get the job ticket which should be the parent ticket
         current_app.logger.error("**************JStor Transformer: Do Task**************")
         current_app.logger.error("WORKER NUMBER " + str(os.getenv('CONTAINER_NUMBER')))
 
+        harvestdir = "/tmp/JSTORFORUM/harvested/loebmusic"
```
[the remark this answers is not in the document]
Comment (michael-lts) [reply app/jstor_transformer.py:46]: Yes, this is configured as part of a pr coming today

File: tasks/tasks.py (hunk 2 above)
Comment (michael-lts) [reply tasks/tasks.py:23]: Yes, this has been removed in a subsequent pr

**Pull request merged**